index.html

```diff
@@ -254,16 +254,19 @@ <h4 class="more_det1 more-de2">LET’S CONNECT</h4>
     <div class="contact-desc">
       If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it 
     </div>
-    <form action="https://formspree.io/f/xbjwdrzl" method="post">
-      <ul class="contact-form">
+    <form novalidate action="https://formspree.io/f/xbjwdrzl" method="post" id="form-con">
+      <ul class="contact-form" id="contact-form">
         <li>
-          <input type="text" id="name" name="user_name" maxlength="30" placeholder="Username" required>
+          <input type="text" class="invalid" id="name" name="user_name" maxlength="30" placeholder="Username" required maxlength="30">
+          <span class="error-hint hidden">Name is required</span>
         </li>
         <li>
-          <input type="email" id="email" name="user_email" placeholder="abc@gmail.com" required>
+          <input type="email" class="invalid" id="email" name="user_email" placeholder="abc@gmail.com" required>
+          <span class="error-hint hidden">Email is required</span>
```

Comment:
Hello @karam084, you have done a tremendous job validating the form fields. The email is well validated, but the message the user gets when they enter an email in capital letters is misleading. I think it would be nice if the feedback they got is to enter an email in lowercase letters 😄. Can you please make that change?